examples/pipeline_wavernn/wavernn_inference_wrapper.py

```diff
@@ -21,18 +21,12 @@
 # *****************************************************************************
 
 
-from typing import List
-
 from torchaudio.models.wavernn import WaveRNN
 import torch
-import torch.nn.functional as F
 import torchaudio
```

Comment:
Not sure if it follows Google's style check. I need your opinion on this to help me understand it.

```
import torch
from torch import Tensor
import torchaudio
from torchaudio.models.wavernn import WaveRNN
```
Or it's like 
```
import torch
import torchaudio
from torch import Tensor
from torchaudio.models.wavernn import WaveRNN
```

Reply:
This part I am not sure since it doesn't trigger flake8's warning.
@mthrok what is our style for sorting the imports?

Reply:
Currently, we do not have a style for sorting imports for historical reason.
So, we resorted to just make sure flake8 does not complain.

Probably it's a good time to start linting as a part of https://fburl.com/jjsby41j